testflight/devices_test.go

```diff
@@ -0,0 +1,24 @@
+package testflight_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	"github.com/onsi/gomega/gbytes"
+)
+
+var _ = Describe("Devices", func() {
+	It("mounts the /dev/fuse device to privileged containers", func() {
+		setAndUnpausePipeline("fixtures/devices.yml")
+
+		watch := fly("trigger-job", "-j", inPipeline("check-fuse-privileged"), "-w")
+		Expect(watch).To(gbytes.Say("succeeded"))
+	})
+
+	It("does not mount the /dev/fuse device to unprivileged containers", func() {
```

Comment:
I can't seem to find the test for unprivileged containers in the test [logs](https://ci.concourse-ci.org/builds/169184268). The privileged one shows up fine.

Reply:
Maybe it just ran faster - they're only logged when the test takes longer than 15 seconds: https://github.com/concourse/ci/blob/bc25c1f71c2e421cfe0e96efca748ce7b4327f3a/tasks/scripts/testflight#L15